examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,413 @@
+                <WalletConnectionTypeButton
+                    buttonStyle={ButtonStyle}
+                    disabledButtonStyle={ButtonStyleDisabled}
+                    connectorType="BrowserWallet"
+                    connectorName="Browser Wallet"
+                    setWaitingForUser={setWaitingForUser}
+                    {...props}
+                />
+                <WalletConnectionTypeButton
+                    buttonStyle={ButtonStyle}
+                    disabledButtonStyle={ButtonStyleDisabled}
+                    connectorType="WalletConnect"
+                    connectorName="Wallet Connect"
+                    setWaitingForUser={setWaitingForUser}
+                    {...props}
+                />
+            </div>
+            <div>
+                {activeConnectorError && <p style={{ color: 'red' }}>Connector Error: {activeConnectorError}.</p>}
+                {!activeConnectorError && !isWaitingForTransaction && activeConnectorType && !activeConnector && (
+                    <p>
+                        <i>Loading connector...</i>
+                    </p>
+                )}
+                {!activeConnection && !isWaitingForTransaction && activeConnectorType && activeConnector && (
+                    <p>
+                        <button style={ButtonStyle} type="button" onClick={connect}>
+                            {isActiveConnectorWaitingForUser && 'Connecting...'}
+                            {!isActiveConnectorWaitingForUser &&
+                                activeConnectorType === 'BrowserWallet' &&
+                                'Connect Browser Wallet'}
+                            {!isActiveConnectorWaitingForUser &&
+                                activeConnectorType === 'WalletConnect' &&
+                                'Connect Mobile Wallet'}
+                        </button>
+                    </p>
+                )}
+                {activeConnectedAccount && (
+                    <>
+                        <div className="text">Connected to</div>
+                        <button
+                            className="link"
+                            type="button"
+                            onClick={() => {
+                                window.open(
+                                    `https://testnet.ccdscan.io/?dcount=1&dentity=account&daddress=${activeConnectedAccount}`,
+                                    '_blank',
+                                    'noopener,noreferrer'
+                                );
+                            }}
+                        >
+                            {activeConnectedAccount}
+                        </button>
+                        <br />
+                        <br />
+                        <div className="containerSpaceBetween">
+                            <button
+                                style={
+                                    activeConnectedAccount === undefined || isRegisterFilePage === false
```

Comment:
```suggestion
                                    !isRegisterFilePage
```

`activeConnectedAccount` is never undefined in this block and `isRegisterFilePage` is always a bool.

This also applies below (also the next button).